Do not use preview api versions

michael-genau opened the pull request:
Use api versions 7.2 as the latest on-premises Azure Dev Ops version does not support these preview api versions.

bryanbcook commented:
Congrats on being my first contribution to this project!

I don't have a way to test against different Azure DevOps server instances, and honestly hadn't considered the use case, according to [this reference ](https://learn.microsoft.com/en-us/rest/api/azure/devops/?view=azure-devops-rest-7.2#api-and-tfs-version-mapping), 7.1 is the latest supported on premises version.

I need to add more integration tests before I can publish on merge, but can I assume you ran the `Test-ExtensionLocaly.ps`?

michael-genau commented:
I executed 'npm run test' to check all the existing test cases and I created the extension via 'npx tfx-cli extension create' to install it on our server.

So far it's working well.

The script Test-ExtensionLocally.ps1 also works.

7.2 was a failure which Azure Dev Ops returned directly. Version 7.1 is correct.
That's why I updated my fork again this afternoon.
In the past I adjusted the code several times and didn't pay attention to the 1.
